src/styles/blocks/informer.css

```
@@ -233,3 +233,103 @@
         top: 15px;
     }
 }
+
+/* Animate */
+
+@keyframes sideSlidesTop {
```

Я бы использовал нейминг ближе к CSS:
```suggestion
@keyframes side-slides-top {
```